src/ImageSharp/Formats/Jpeg/Components/Encoder/LuminanceForwardConverter{TPixel}.cs

```diff
@@ -15,39 +16,63 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.Components.Encoder
     internal ref struct LuminanceForwardConverter<TPixel>
         where TPixel : unmanaged, IPixel<TPixel>
     {
+        /// <summary>
+        /// Number of pixels processed per single <see cref="Convert(int, int, ref RowOctet{TPixel})"/> call
+        /// </summary>
+        private const int PixelsPerSample = 8 * 8;
+
+        /// <summary>
+        /// <see cref="Size"/> of sampling area from given frame pixel buffer.
+        /// </summary>
+        private static readonly Size SampleSize = new Size(8, 8);
```

Comment:
`Size` is simple struct (just containing 2 ints), so it may be better to just use a property / factory here:
```suggestion
        private static Size SampleSize => new Size(8, 8);
```
so for each access a new struct is created, but as it's small the JIT could enregister and perform other optimizations, that aren't possible by reading the struct from memory.

In short with the property we
* can avoid a memory read
* allow the JIT to perform optimizations

The following benchmark is _not_ the most representative one, but (at least for .NET 6 and I believe starting from .NET Core 3.1) one can see where it goes by looking at the disassembly.
<details>
  <summary>Benchmark code</summary>

```c#
using BenchmarkDotNet.Attributes;

BenchmarkDotNet.Running.BenchmarkRunner.Run<Bench>();

[DisassemblyDiagnoser]
[ShortRunJob]
public class Bench
{
    private static readonly Size s_samplesize = new(8, 8);
    private static Size SampleSize => new(8, 8);

    [Benchmark(Baseline = true)]
    public int StaticReadonly() => s_samplesize.Width + s_samplesize.Height;

    [Benchmark]
    public int Property() => SampleSize.Width + SampleSize.Height;
}

public struct Size
{
    public int Width  { get; set; }
    public int Height { get; set; }

    public Size(int width, int height) => (this.Width, this.Height) = (width, height);
}
```
</details>

```assembly
; .NET 6.0.0 (6.0.21.52210), X64 RyuJIT

; Bench.StaticReadonly()
       mov       rax,25118002F10
       mov       rax,[rax]
       mov       edx,[rax+8]
       add       edx,[rax+0C]
       mov       eax,edx
       ret
; Total bytes of code 22

; Bench.Property()
       mov       eax,10
       ret
; Total bytes of code 6
```

With the property the applies all possible optimizations, while with the `static readonly` it must read from memory and can't do optimizations.

Reply:
I didn't even know jit ca do this with small structs, thanks!